runtimes/workspace/versions/latest/src/server.js

```diff
@@ -0,0 +1,320 @@
+
+const synapse = new Synapse("localhost", 3000, workdir);
+let terminal, filesystem, system, git, code; // initialize services
+
+const router = {
+  synapse: async (message) => {
+    const { operation, params } = message;
+    let result;
+
+    switch (operation) {
+      case "updateWorkDir":
+        result = await synapse.updateWorkDir(params.workdir);
+        break;
+      default:
+        throw new Error("Invalid synapse operation");
+    }
+    return result;
+  },
+
+  terminal: async (message) => {
+    const { operation, params } = message;
+
+    switch (operation) {
+      case "updateSize":
+        terminal.updateSize(params.cols, params.rows);
+        break;
+      case "createCommand":
+        terminal.createCommand(params.command);
+        break;
+      default:
+        throw new Error("Invalid terminal operation");
+    }
+    return null;
+  },
+
+  fs: async (message) => {
+    const { operation, params } = message;
+    let result;
+
+    switch (operation) {
+      case "createFile":
+        result = await filesystem.createFile(params.filepath, params.content);
+        break;
+      case "getFile":
+        result = await filesystem.getFile(params.filepath);
+        break;
+      case "updateFile":
+        result = await filesystem.updateFile(params.filepath, params.content);
+        break;
+      case "updateFilePath":
+        result = await filesystem.updateFilePath(
+          params.filepath,
+          params.newPath,
+        );
+        break;
+      case "deleteFile":
+        result = await filesystem.deleteFile(params.filepath);
+        break;
+      case "createFolder":
+        result = await filesystem.createFolder(params.folderpath);
+        break;
+      case "getFolder":
+        result = await filesystem.getFolder(params.folderpath);
+        break;
+      case "updateFolderName":
+        result = await filesystem.updateFolderName(
+          params.folderpath,
+          params.name,
+        );
+        break;
+      case "updateFolderPath":
+        result = await filesystem.updateFolderPath(
+          params.folderpath,
+          params.newPath,
+        );
+        break;
+      case "deleteFolder":
+        result = await filesystem.deleteFolder(params.folderpath);
+        break;
+      default:
+        throw new Error("Invalid operation");
+    }
+    return result;
+  },
+
+  system: async (message) => {
+    const { operation } = message;
+    let result;
+
+    switch (operation) {
+      case "getUsage":
+        result = await system.getUsage();
+        break;
+      default:
+        throw new Error("Invalid system operation");
+    }
+    return result;
+  },
+
+  git: async (message) => {
+    const { operation, params } = message;
+    let result;
+
+    switch (operation) {
+      case "init":
+        result = await git.init();
+        break;
+      case "addRemote":
+        result = await git.addRemote(params.name, params.url);
+        break;
```

Comment:
```suggestion
      case "init":
        return git.init();
      case "addRemote":
        return git.addRemote(params.name, params.url);
```

no need for the result param 👍🏻